.pre-commit-config.yaml

```diff
@@ -1,97 +1,95 @@
--   repo: local
+  - repo: local
     hooks:
-#    - id: releaser
-#      name: releaser
-#      entry: python ./misc/releaser.py check
-#      language: python
-#      language_version: python3
-#      pass_filenames: false
-#      require_serial: true
-    - id: license_headers
-      name: license_headers
-      entry: python ./misc/license_headers.py add
-      language: python
-      language_version: python3
-      # see: https://github.com/pre-commit/identify/blob/master/identify/extensions.py
-      types_or: [python, sql, rust, vue, ts, tsx, javascript, jsx]
+      #    - id: releaser
+      #      name: releaser
+      #      entry: python ./misc/releaser.py check
+      #      language: python
+      #      language_version: python3
+      #      pass_filenames: false
+      #      require_serial: true
+      - id: license_headers
+        name: license_headers
+        entry: python ./misc/license_headers.py add
+        language: python
+        language_version: python3
+        # see: https://github.com/pre-commit/identify/blob/master/identify/extensions.py
+        types_or: [python, sql, rust, vue, ts, tsx, javascript, jsx]
 
-# Python
+  # Python
 
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: 19.3b0
     hooks:
-    - id: black
-      additional_dependencies: ['click==8.0.4']
-      args:
+      - id: black
+        args:
           - "--line-length=100"
           - "--exclude=parsec/core/gui/_resources_rc.py"
           - "--exclude=parsec/core/gui/ui/"
-      language_version: python3
-      additional_dependencies: ['click==8.0.4']
+        language_version: python3
+        additional_dependencies: ["click==8.0.4"]
 
--   repo: https://gitlab.com/pycqa/flake8
+  - repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.3
     hooks:
-    - id: flake8
+      - id: flake8
 
--   repo: https://github.com/pre-commit/mirrors-mypy
+  - repo: https://github.com/pre-commit/mirrors-mypy
     rev: v0.812
     hooks:
-    - id: mypy
-      additional_dependencies: ['trio_typing==0.5.1', 'pendulum==2.1.2']
-      # Mypy will complain if it find two files with the same name which
-      # are not part of a package (typically occurs for packaging script helpers)
-      files: ^(parsec|tests)/
+      - id: mypy
+        additional_dependencies: ["trio_typing==0.5.1", "pendulum==2.1.2"]
+        # Mypy will complain if it find two files with the same name which
+        # are not part of a package (typically occurs for packaging script helpers)
+        files: ^(parsec|tests)/
 
-# Rust
+  # Rust
 
--   repo: https://github.com/doublify/pre-commit-rust
+  - repo: https://github.com/doublify/pre-commit-rust
     rev: v1.0
     hooks:
-    - id: fmt
-    - id: clippy
-      args: [
-        --workspace,
-        --tests,
-        --bins,
-        --lib,
-        --,
-        --deny=warnings,
-        --deny=clippy::self_named_module_files
-      ]
+      - id: fmt
+      - id: clippy
+        args:
```

Comment:
not sure this `- --...` form is more readable 🤔

Reply:
It was to normalize with the `args` of `black`.